docs/source/terms_and_model.rst

```diff
@@ -379,9 +375,49 @@ A contiguous region specified by named features.
    {
      'end': 'q22.3',
      'start': 'q22.2',
-     'type': 'NamedInterval'
+     'type': 'CytobandInterval'
    }
 
+.. _TextInterval:
+
+TextInterval
+$$$$$$$$$$$$$$
+
+**Computational definition**
+
+A contiguous region specified by named features.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - type
+     - string
+     - 1..1
+     - Interval type; MUST be set to '**TextInterval**'
+   * - start
+     - string
+     - 1..1
+     - name of feature start
+   * - end
+     - string
+     - 1..1
+     - name of feature end
+
+**Implementation guidance**
+
+* `start` and `end` attributes of TextInterval are intentionally
+  specified vaguely in order to accommodate a wide variety of
+  uses.
+
```

Comment:
An example of a TextInterval as given for SimpleInterval and CytobandInterval would be useful.